### ISSUE TITLE
Widget popout does not work in electron

### ISSUE THREAD

**Issue by GintV** [iTwin/appui#402]
**Describe the bug**
Widget popout does not work.

**To Reproduce**

1. Define simplest widget possible that has a `canPopout` porperty set to true. 
3. Click the popout button.

**Actual behavior**

![image](https://github.com/iTwin/appui/assets/17436829/5c656a8b-c491-43ac-a7df-5e09e30de6ea)
![image](https://github.com/iTwin/appui/assets/17436829/72b53b2b-392e-418d-a40c-8ae2aa994416)

**Expected behavior**
It should work.

**Desktop (please complete the applicable information):**

- OS: win 10/11
- Electron 24.1.2 
- iTwin.js AppUI Version 4.2.0
- iTwin.js Version 4.0.2

**Comment by aruniverse** [comment]
@a-gagnon didnt civil run into this? How did you guys fix this?

**Comment by a-gagnon** [comment]
> @a-gagnon didnt civil run into this? How did you guys fix this?

Correct. We had a window open handler trying to open links in the system browser instead of electron.
Removed the following code solved the issue:
```ts
  // open links in the system browser instead of Electron
  // remove this if you desire the default behavior instead
  ElectronHost.mainWindow.webContents.setWindowOpenHandler(({ url }) => {
    void shell.openExternal(url);
    return { action: "deny" };
  });

**Comment by GintV** [comment]
@a-gagnon @aruniverse had the same issue and fixed it like that as well. After that, as you can see, new browser windows popups without content. At that time thought that it might be because we're still on UI1.0. After switching to UI2.0 and updating all the packages it still didn't work.

**Comment by GerardasB** [comment]
Do you deliver `iTwinPopup.html` page along side `index.html`?
Have you looked at https://github.com/iTwin/appui/blob/master/test-apps/appui-test-app/standalone/src/backend/electron/ElectronMain.ts?
I see no problems with popout widgets in our own test app: https://github.com/iTwin/appui/tree/master/test-apps/appui-test-app/standalone Have you tried running that and compared it to your application?

**Comment by GintV** [comment]
Cannot tell, I filed issue on behalf of @arnoldas-v since he's on vacation for 2 more weeks. One thing I didn't mentioned in a bug is errors we're seeing in poped out window:

![image](https://github.com/iTwin/appui/assets/17436829/2cde08c8-53f1-40e5-8992-df14881413f1)

@NancyMcCallB asked to file an issue. I understood that there's some kind of problem in appui and by editing appui under node_modules @arnoldas-v got it working.

**Comment by GerardasB** [comment]
The error indicates that the opened window is running in a separate JS context (i.e. in the main window Redux store is already initialized). We can wait for @arnoldas-v to give us more details.

**Comment by raplemie** [comment]
Hi, a fix for this was provided and released in 4.3.0. Let us know if that resolves your issue!
